src/router/routes/errorHandling.ts

```diff
@@ -28,7 +34,7 @@ export const applicationErrorMapper = (
         next(new createError.NotFound(err.message));
         return;
     }
-    if (isInvalidResourceError(err)) {
+    if (isInvalidResourceError(err) || isBadRequestError(err)) {
```

Comment:
If isInvalidResourceError is treated the same way as isBadRequestError from the customers' perspective, what's the gain of differentiating the two?

Reply:
It would be a bit strange to use isInvalidResourceError for all bad request errors, I think BadRequestError and we can deprecate isInvalidResourceError later